### ISSUE TITLE
[Improvement] Run test in parallel

### ISSUE THREAD

**Issue by yuqi1129** [datastrato/gravitino#1198]
### What would you like to be improved?

We can run all tests simultaneously to reduce testing time.

**Comment by yuqi1129** [comment]
> there's more work about integrating tests, especially for IcebergRESTService test.

Agreed, things are not that simple. Some tests cannot be run parallelly. For example, the Integration tests can be run at the same time if we only add the `maxParallelForks` configuration item.

**Comment by jerryshao** [comment]
The CI environment only has one core, it is not so useful to increase the parallelism.
